services/api/blocksim_ratelimiter.go

```diff
@@ -110,6 +119,49 @@ func (b *BlockSimulationRateLimiter) CurrentCounter() int64 {
 	return atomic.LoadInt64(&b.counter)
 }
 
+func SendHTTPRequest(client *http.Client, req *common.BuilderBlockValidationRequest, url string, headers http.Header) (requestErr, validationErr error) {
+	payloadBytes, err := req.MarshalSSZ()
+	if err != nil {
+		return fmt.Errorf("could not marshal request: %w", err), nil
+	}
+	httpReq, err := http.NewRequest(http.MethodPost, url, bytes.NewReader(payloadBytes))
+	if err != nil {
+		return fmt.Errorf("invalid request for %s: %w", url, err), nil
+	}
+
+	httpReq.Header.Add("Content-Type", "application/octet-stream")
+	httpReq.Header.Add("Eth-Consensus-Version", strings.ToLower(req.Version.String()))
```

Comment:
allow future forks to reuse the same endpoint instead of a new endpoint incremented by version number, like in the JSON RPC endpoint.